### PR TITLE
Enable GitHub actions

### LINKED ISSUE #92: release pre-built binaries (using GitHub Actions, perhaps)
@cyphar WDYT?

### PR DESCRIPTION
**Pull request by AkihiroSuda**
Fix #92

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
Not triggered?

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  artifact:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - run: DOCKER_BUILDKIT=1 docker build -o /tmp/artifact --target artifact -f Dockerfile.buildtests .
+    - run: (cd /tmp/artifact; sha256sum *)
+    - uses: actions/upload-artifact@v1
+      with:
+        name: slirp4netns-x86_64
+        path: /tmp/artifact/slirp4netns
```
Comment (AkihiroSuda) [inline .github/workflows/main.yaml]: note: this becomes zip :(

https://github.com/actions/upload-artifact/issues/3

**Comment by AkihiroSuda** [comment]
@rootless-containers/slirp4netns-approvers PTAL

**Comment by giuseppe** [comment]
I am not familiar with GH actions, but let's give it a try :-)

LGTM

**Pull request merged**